### ISSUE TITLE
Nginx does not start after system reboot

### ISSUE THREAD

**Issue by StefanRother-OTOBO** [RotherOSS/otobo-docker#1]
It's a known issue. We need to fix it with a workarround.

**Comment by bschmalhofer** [comment]
I have added the missing `restart: always` declarations for `otobo_cron_1`and `otobo_nginx_1`. Can you retry with 75c1530296b055931390c6dd68694da5639b3b88 ?

Worked for me. I started the containers, rebooted the host and started the docker daemon:

```
bernhard@bernhard-HP-250-G6-Notebook-PC:~/devel/OTOBO/otobo-docker$ docker-compose ps
     Name                    Command               State                    Ports                  
---------------------------------------------------------------------------------------------------
otobo_cron_1      bin/docker/entrypoint.sh cron    Up                                              
otobo_db_1        docker-entrypoint.sh --max ...   Up      3306/tcp                                
otobo_elastic_1   /tini -- /usr/local/bin/do ...   Up      9200/tcp, 9300/tcp                      
otobo_nginx_1     /docker-entrypoint.sh ngin ...   Up      0.0.0.0:443->443/tcp, 0.0.0.0:81->80/tcp
otobo_redis_1     docker-entrypoint.sh redis ...   Up      6379/tcp                                
otobo_web_1       bin/docker/entrypoint.sh web     Up    


```

**Comment by StefanRother-OTOBO** [comment]
Hi,

I tested it not again, but I think it does not work for Nginx. It's a known error. 
Did you test it and did it work for you?

Thanks, Stefan

**Comment by bschmalhofer** [comment]
> Did you test it and did it work for you?
> 
https://github.com/RotherOSS/otobo-docker/commit/75c1530296b055931390c6dd68694da5639b3b88 worked for me. The fix is included  in OTOBO 10.0.2. But I'll check again later this day.

For testing with a local devel image:

1. git clone git@github.com:RotherOSS/otobo.git
2. cd otobo
2. git checkout rel-10_0
2. git pull
2. bin/docker/build_docker_images.sh
2. Go to the otobo-doker dir and set `OTOBO_IMAGE_OTOBO_NGINX=otobo-nginx-webproxy` 
2. test

@svenoe  : Could you check whether this works for you with OTOBO 10.0.2 ?

**Comment by bschmalhofer** [comment]
Checked the reboot behavior with the local-10.0.x, that is pre 10.0.4, images and it looked good. Closing the issue.
